lib/renderer/init.js

```diff
@@ -116,6 +116,12 @@ if (nodeIntegration === 'true') {
   } else {
     global.__filename = __filename
     global.__dirname = __dirname
+
+    if (window.location.protocol !== 'chrome-devtools:') {
+      // Search for module under the app directory
+      // (remote.app doesn't work in devtools)
+      module.paths = module.paths.concat(Module._nodeModulePaths(electron.remote.app.getAppPath()))
```

Comment:
`remote.app.getAppPath()` is a synchronous IPC call which would block the renderer process, could this be done instead with a command line flag like opener id or guest instance id is done?